src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -30,6 +30,13 @@ public class ConfigurationManager<T> : BaseConfigurationManager, IConfigurationM
         private T _currentConfiguration;
         private Exception _fetchMetadataFailure;
         private TimeSpan _bootstrapRefreshInterval = TimeSpan.FromSeconds(1);
+        private static readonly DistributedConfigurationOptions s_distributedConfigurationOptions = new();
+
+        // L2 TODO: internal until L2 cache is implemented S2S.
+        /// <summary>
+        /// TODO inject via ctors
+        /// </summary>
+        internal IDistributedConfigurationManager<T> DistributedConfigurationManager { get; set; }
```

Comment:
How will the user set this property (after it is public)?

Reply:
I think that this property will be set via the ctors, does that sound right @GeoK?